scripts/vagrant/multi/stop_vagrant.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+set -xe
+
+export BOX="ubuntu/xenial64"
+PROVIDER=${PROVIDER:-virtualbox}
+
+if [[ "$PROVIDER" != "virtualbox" ]]; then
+    if [[ "$USER" == "" ]]; then
+        echo "USER env var not set"
+        exit 1
+    fi
+    if [[ "$SSH_KEY" == "" ]]; then
+        echo "SSH_KEY env var not set"
+        exit 1
+    fi
+fi
```

Comment:
Does this need to be added to `/single/`? Or maybe we can reuse the same script for both?